src/components/Header.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+    <header>
+        <img src="../assets/postit.png">La Postiterie
+    </header>
+</template>
+
+<script>
+export default {
+    name: 'Header'
+}
+</script>
+
+<style scoped>
+
+header {
+    display:flex;
+    justify-content: center;
+    align-items: center;
+    font-size : 1.8em;
+    text-align:center;
+    font-weight: 900;
+    background-color: #478BF9;
```

Comment:
Can we externalize these kind of variables to a theme? So that if I choose to change theme colors I don't have to change in every file using it, but only once.